packages/foam-vscode/src/core/janitor/convert-links-format.ts

```diff
@@ -81,21 +79,17 @@ export function convertLinkFormat(
     if (relativeUri.getBasename() === resource.uri.getBasename()) {
       url = '';
     }
-    if (sectionDivider === '#') {
-      url = `${url}${sectionDivider}${section}`;
-    }
-    if (url.indexOf(' ') > 0) {
-      url = `<${url}>`;
-    }
 
     /* if it's originally an embedded note, the markdown link shouldn't be embedded */
-    if (embed && targetRes.type === 'note') {
-      embed = '';
-    }
-    return {
-      newText: `${embed}[${alias}](${url})`,
-      range: link.range,
-    };
+    const isEmbed =
+      link.isEmbed && targetRes.type === 'note' ? false : undefined;
```

Comment:
I guess what you are saying is that only images can be embedded?

```suggestion
      targetRes.type === 'image' ? link.isEmbed : false;
```